src/components/TabSelector.js

```diff
@@ -0,0 +1,62 @@
+import {View} from 'react-native';
+import React from 'react';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import * as Expensicons from './Icon/Expensicons';
+import ONYXKEYS from '../ONYXKEYS';
+import TabSelectorItem from './TabSelectorItem';
+import Tab from '../libs/actions/Tab';
+import CONST from '../CONST';
+import useLocalize from '../hooks/useLocalize';
+import styles from '../styles/styles';
+import * as IOU from '../libs/actions/IOU';
+
+const propTypes = {
+    /** Which tab has been selected */
+    tabSelected: PropTypes.string,
+
+    /** The current money request ID */
+    moneyRequestID: PropTypes.string,
+};
+
+const defaultProps = {
+    tabSelected: CONST.TAB.TAB_MANUAL,
+    moneyRequestID: '',
+};
+
+function TabSelector(props) {
```

Comment:
I know we have a few more places across the app that will soon be using this same UI pattern, so I think it's a good idea to generalize this component a bit.